astro/src/content/docs/get-started/start-here/index.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: "Getting Started with FusionAuth: A Beginner's Guide"
+description: Learn how to set up FusionAuth, integrate authentication into your app, and implement advanced features like magic link logins in this developer-friendly guide.
+section: get started
+subcategory: start here
+navOrder: 0
+codeRoot: https://raw.githubusercontent.com/synedra/fusionauth-first-steps-launch/main/
+
+---
+import ListHostedLoginPagesUseCases from 'src/content/docs/_shared/_list-hosted-login-pages-use-cases.mdx';
+import StartDiagram from 'src/diagrams/docs/get-started/start-here/launch-delegate-add.astro';
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+import Aside from 'src/components/Aside.astro';
+import InlineField from 'src/components/InlineField.astro';
+import InlineUIElement from 'src/components/InlineUIElement.astro';
+import DockerComposeFiles from 'src/content/docs/get-started/download-and-install/_docker-compose-files.mdx';
+import { YouTube } from '@astro-community/astro-embed-youtube';
+import DockerSpinup from 'src/components/quickstarts/DockerSpinup.astro';
```

Comment:
is this the wrong component?

Reply:
I am using the same component as the Quickstart, because it is correct in the get-started case as well.  I could make a copy and move that to get-started